apps/dev-portal-web/domains/docs/components/AmountDistributionCalculator.tsx

```diff
@@ -0,0 +1,357 @@
+import { faker } from '@faker-js/faker'
+import { Col, Row, RowProps } from 'antd'
+import React, { useCallback, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { Plus, Trash } from '@open-condo/icons'
+import { Alert, Button, Card, Input, List, MarkdownCodeWrapper, Switch, Tabs, Typography } from '@open-condo/ui'
+
+import { split, createRecipientKey } from '@condo/domains/acquiring/utils/billingCentrifuge'
+
+type TRecipient = {
+    tin: string
+    bic: string
+    bankAccount: string
+}
+
+type TSplit = {
+    recipient: TRecipient | null
+    amount?: string
+    feeAmount?: string
+}
+
+type TDistributionItem = {
+    recipient: TRecipient
+    amount: string
+    isFeePayer?: boolean
+    order?: number
+    vor?: boolean
+    overpaymentPart?: number
+}
+
+type TSplitOptions = {
+    appliedSplits?: TSplit[]
+    decimalPlaces?: number
+    feeAmount?: string
+}
+
+type TDistributionItemProps = {
+    index: number
+    item: TDistributionItem
+    onUpdate: (index: number, item: TDistributionItem) => void
+}
+
+const PADDING = 8
+const GUTTER_COMPACT_ROW: RowProps['gutter'] = [PADDING, PADDING]
+const GUTTER_ROW: RowProps['gutter'] = [16, 16]
+
+const DistributionItem: React.FC<TDistributionItemProps> = (props) => {
+    const { index, item, onUpdate } = props
+
+    const intl = useIntl()
+
+    const TinTitle = intl.formatMessage({ id: 'amountDistributionCalculator.tin' })
```

Comment:
incorrect translation keys format

Reply:
`docs.widgets.<name>.*`

Reply:
The component was moved to docs/widgets, so translation keys were updated too.